### PR TITLE
Who We Are Page Fixes

### PR DESCRIPTION
**Pull request by seanbsullivan**
Fixed font sizing across hero, contentimagefull, and the buttons. Fixed image sizes and added the actual images from figma. After this is merged I'll work on adding the bottom 2 sections that are missing from the About page (Mas calls it the "Who We Are" page in his notes)

### CODE REVIEW

File: src/components/elements/Hero.svelte [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-    <h1 class="block text-4xl font-bold md:text-5xl lg:text-6xl {titleClasses}">
+    <h1 class="block font-bold text-xl md:text-[40px] {titleClasses}">
       {title}
     </h1>
   </div>
   <!-- End Title -->
 
   <div class="max-w-3xl mx-auto mt-5 text-center">
-    <p id="content" class="text-white {contentClasses}">
+    <p id="content" class="text-white {contentClasses} md:text-[24px]">
```
Comment (dfullaway) [inline src/components/elements/Hero.svelte]: 24px is 2xl. Also, I think if you put the text size here, you won't be able override with contentClasses.

File: src/pages/about.astro [hunk 2]
```diff
@@ -16,43 +16,50 @@ import FooterFull from '../components/footer/FooterFull.svelte'
...
   >
     <div
-      class="relative z-20 h-screen py-20 snap-start bg-[url('/images/marine.png')] bg-cover bg-no-repeat"
+      class="relative z-20 h-screen py-20 bg-[url('/images/marine.png')] bg-cover bg-no-repeat"
     >
       <Hero
         padding="pt-24"
         title="WHO WE ARE"
         content="The Marine Corps Software Factory welcomes Marines from any MOS who have the aptitude, competence, and discipline to serve in key roles on software development teams."
+        contentClasses="text-2xl"
```
Comment (dfullaway) [inline src/pages/about.astro]: This breaks the mobile font size, and you modified Hero to set this size anyway. Either remove entirely and have the default hero font size be set or use md:text-2xl.

File: src/pages/about.astro [hunk 3]
```diff
@@ -16,43 +16,50 @@ import FooterFull from '../components/footer/FooterFull.svelte'
     <img src="/ega.svg" />
   </div>
   <div
-    class="relative max-h-screen overflow-y-scroll font-expressway red-radial-gradient scrollbar-hide snap-start snap-y snap-mandatory"
+    class="relative max-h-screen overflow-y-scroll font-expressway red-radial-gradient scrollbar-hide"
```
Comment (dfullaway) [inline src/pages/about.astro:19]: I think you will probably need to change the horizontal size away from max-h-screen, which was used to break up the page for the snapping. The extra space doesn't line up with the figma.

**Review by dfullaway — COMMENTED** [review]
Just a couple of small changes to get this ready to pull.

File: src/components/sections/ContentImageFull.svelte [hunk 4]
```diff
@@ -1,59 +1,60 @@
...
+          <RedButton href={link} text={linkText} />
+        </div>
       {/if}
     </div>
-  </section>
\ No newline at end of file
+    {#if !flipped}
+      <div class="grid gap-4 mt-8 grid-cols-1">
+        <img
+          class="w-full rounded-lg aspect-square object-cover {imageClasses}"
```
Comment (wcrum) [inline src/components/sections/ContentImageFull.svelte]: Can you change this to either w-3/4 or w-7/8. Then the scope needs to be

```html
<div class="flex justify-center">
  <img
```

Comment (seanbsullivan) [reply src/components/sections/ContentImageFull.svelte]: change done now

File: src/pages/about.astro (hunk 2 above)
Comment (seanbsullivan) [reply src/pages/about.astro]: fixed thanks

File: src/pages/about.astro (hunk 3 above)
Comment (seanbsullivan) [reply src/pages/about.astro:19]: updated and fixed spacing

File: src/components/elements/Hero.svelte (hunk 1 above)
Comment (seanbsullivan) [reply src/components/elements/Hero.svelte]: changed to 2xl and moved so it can be overridden

File: src/pages/about.astro [hunk 5]
```diff
@@ -19,47 +19,84 @@ import FooterFull from '../components/footer/FooterFull.svelte'
...
-        linkText="LEARN MORE"
-        title="OUR IMPACT"
-        content="By providing Marines with the skills to develop software and implement artificial intelligence, we are giving commanders the ability to quickly produce software and AI solutions at the tactical edge."
-        image="images/marine_whiteboard.jpeg"
-        imageClasses="md:h-[550px] md:w-[550px] md:min-w-[550px]"
-      />
-    </div>
-    <div class="relative z-20 pt-20">
-      <FooterFull client:load />
+    <div class="lg:hidden">
```
Comment (dfullaway) [inline src/pages/about.astro]: This breakpoint should probably be md to match when the ContentImagFull swaps to mobile view.

**Review by dfullaway — COMMENTED** [review]
Just one small change, and I think it will be good.

File: src/pages/about.astro (hunk 5 above)
Comment (seanbsullivan) [reply src/pages/about.astro]: good catch, fixed now

**Pull request merged**